### PR TITLE
NP-581:library updates

### PR DESCRIPTION
**Pull request by axthosarouris**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -79,35 +76,35 @@ repositories {
...
 
     //this should be the same version as the one used by amazon
-    implementation ("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.6.0")
+    implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.10.3")
 
     // useful java libraries
-    implementation ("com.google.guava:guava:27.1-jre")
+    implementation("com.google.guava:guava:27.1-jre")
 
     implementation('io.symphonia:lambda-logging:1.0.3')
```
Comment (janbj) [inline build.gradle]: We should probably just use System.out for logging

File: build.gradle [hunk 2]
```diff
@@ -79,35 +76,35 @@ repositories {
...
-    implementation ("com.amazonaws:aws-java-sdk-logs:${awsSdkVersion}")
+    implementation("com.amazonaws:aws-java-sdk-logs:${awsSdkVersion}")
 
     //this should be the same version as the one used by amazon
-    implementation ("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.6.0")
+    implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.10.3")
 
     // useful java libraries
-    implementation ("com.google.guava:guava:27.1-jre")
+    implementation("com.google.guava:guava:27.1-jre")
```
Comment (janbj) [inline build.gradle]: Still relevant and useful in a Java 11 world?

File: build.gradle (hunk 1 above)
Comment (axthosarouris) [reply build.gradle]: made an issue as a reminder

File: build.gradle (hunk 2 above)
Comment (axthosarouris) [reply build.gradle]: Created an issue as a reminder

File: settings.gradle [hunk 3]
```diff
@@ -0,0 +1,11 @@
+
+plugins {
```
Comment (axthosarouris) [inline settings.gradle]: The com.gradle.build-scan plugin is now called 'com.gradle.enterprise' and must be added in the settings file and not the build file

Comment (janbj) [reply settings.gradle]: What are we using this for?

File: build.gradle [hunk 4]
```diff
@@ -79,35 +82,36 @@ repositories {
...
 
+    // TODO: Should we use System.out instead?
     implementation('io.symphonia:lambda-logging:1.0.3')
 
-    compile ("org.jacoco:org.jacoco.core:0.8.3")
+    compile("org.jacoco:org.jacoco.core:0.8.3")
 
-    //libraries for testing
-    testImplementation ("org.mockito:mockito-core:2.27.0")
+    // libraries    for    testing
```
Comment (janbj) [inline build.gradle]: Isn't that obvious from testImplementation? :)

File: .travis.yml [hunk 5]
```diff
@@ -18,13 +12,4 @@ cache:
   directories:
   - "$HOME/.gradle/caches/"
   - "$HOME/.gradle/wrapper/"
-deploy:
-  provider: script
-  script: "bash ./gradlew bintrayUpload"
-  user: brinxmat
-  key:
-    secure: WuJ9ct4sACFkwYzp30DW+FpcLJjmlruvQm4XrpeFzA5OxGFKFoDT5CrSPE4oHeIMRmZvcyPk4HnjrV4h81KRJ9ZF1VxdG0MIQdyYFU0FaiTuqhT1jT27GuzpuSRvJKz34u/whbQE/l0uPXNb4oDjERNJX2A44wku/dJGXTtyDnyQdElQQINO2L8I9aAM9uj95jt0JrISfNzEK8Ls1t3kjbuh1FGomyCRHvLyU1yl20v3znbeURLW6VjZIfmGB2+SOgQJlY6g+qWstq1OSs4YfSDZpyzRo/SoKV+mrEdtSXN/dylGfxc0DssILYm0mY/T6mMfrRgBUq+ly0ikyXmYWHCxZF6WGLNHWdNlHKFyJwjrill7iWi7IX79St4Yg/50+hgvaitZohl5rdQ0Dm3iecXAYcAtl06JAfLCBwKayuYrCiyaWW1ST+cMUDtBE+x4Yllg/v7+LftNGpW3VDuP75pAcNfoqDax0tGAaMsr/g6eA52tVJ3Y5/409sUvFg1ekdybTBrbzzWhnmycFDVsdo3S+7sQvXdo6tt3BnyVPOAUpjifFxlOfov61jCmYjtOEY4uhZ5k5jRZXBOSGv5lp6nw9ke3MQuoHtnbPbCY0bj9ZqVz3E9Ld5HbK0A4Ffa8nkPNbLmuI8pUe1Of1ityerN036iYC4q4+TAu4ED8NpA=
```
Comment (janbj) [inline .travis.yml]: Great!

**Pull request merged**